dashboard/settings/base.py

```diff
@@ -169,7 +169,15 @@
 
 LOGIN_URL = '/login/'
 LOGIN_REDIRECT_URL = '/'
-
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
```

Comment:
this won't work so well on aws but we can look in to setting up elasticache or something at a later date